README.md

```diff
@@ -10,29 +10,20 @@
 
 ![TruLens](https://www.trulens.org/assets/images/Neural_Network_Explainability.png)
 
-TruLens provides a set of tools for developing and monitoring neural nets,
-including large language models. This includes both tools for evaluation of LLMs
-and LLM-based applications with _TruLens-Eval_ and deep learning explainability
-with _TruLens-Explain_. _TruLens-Eval_ and _TruLens-Explain_ are housed in
-separate packages and can be used independently.
-
-
-## About
-
-**Don't just vibe-check your LLM app!** Systematically evaluate and track your
+**Don't just vibe-check your llm app!** Systematically evaluate and track your
 LLM experiments with TruLens. As you develop your app including prompts, models,
-retrievers, knowledge sources and more, *TruLens* is the tool you need to
+retreivers, knowledge sources and more, *TruLens* is the tool you need to
 understand its performance.
 
 Fine-grained, stack-agnostic instrumentation and comprehensive evaluations help
 you to identify failure modes & systematically iterate to improve your
 application.
 
 Read more about the core concepts behind TruLens including [Feedback Functions](https://www.trulens.org/trulens_eval/getting_started/core_concepts/feedback_functions/),
-[The RAG Triad](https://www.trulens.org/trulens_eval/getting_started/core_concepts/rag_triad/),
-and [Honest, Harmless and Helpful Evals](https://www.trulens.org/trulens_eval/getting_started/core_concepts/honest_harmless_helpful_evals/).
+[The RAG Triad](https://www.trulens.org/trulens/getting_started/core_concepts/rag_triad/),
```

Comment:
Where do we configure the docs route? Will need to move 
`https://www.trulens.org/trulens_eval ` to `https://www.trulens.org/trulens`